basilisk/src/ast/interpreter/dimension-tests/test15.c

```
@@ -0,0 +1,18 @@
+double c = G*H0;
+
+int main()
+{
+  L0 = 1. [1];
+  G = G0;
+  run();
+}
+
+event init (i = 0)
+{
+  foreach()
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If foreach is a macro then please configure it.

(unknownMacro)